### PR TITLE
YOU BUTTERFLY BELIEVE IT

### PR DESCRIPTION
**Pull request by mkirsche**

### CODE REVIEW

File: rec/pokemoninfo.txt [hunk 1]
```diff
@@ -935,6 +935,7 @@ BUG NONE
 40
```
Comment (leahfortier) [inline rec/pokemoninfo.txt]: you need to update this number to 41

**Comment by leahfortier** [comment]
thank you for this! we should make this change for other obvious flyers like Beedrill and whatnot

File: rec/pokemoninfo.txt [hunk 2]
```diff
@@ -70208,4 +70209,4 @@ ATTRACT
 FACADE
 ROAR
 SNARL
-CONFIDE
\ No newline at end of file
+CONFIDE
```
Comment (leahfortier) [inline rec/pokemoninfo.txt]: nit: revert to preserve blame

Comment (mkirsche) [inline rec/pokemoninfo.txt]: I don't know how to git well enough to do that

**Pull request merged**